lib/has_di_parameters.rb

```diff
@@ -48,10 +48,10 @@
 require "active_support"
 require "active_support/core_ext"
 
-module InitializeWithConfig
+module HasDIParameters
```

Comment:
I'm still struggling to see the value of this vs. basic keywords args with defaults and accessors. I think the name is a smell - we essentially have a module called 'has parameters' !

Reply:
I've switched in the areas where it didn't make sense, but left it where we want to change things from the top level. I changed the name to make it more obvious that we're using something more akin to the way that "params" work when passed through from the controller.